.travis.yml

```diff
@@ -26,38 +26,19 @@ after_success:
     - if [ ! -z "$TRAVIS_TAG" ] ; then
         export DOCKER_CLI_EXPERIMENTAL=enabled
 
-        docker tag $DOCKER_NS/gateway:latest-dev $DOCKER_NS/gateway:$TRAVIS_TAG;
         echo $DOCKER_PASSWORD | docker login -u=$DOCKER_USERNAME --password-stdin;
-        docker push $DOCKER_NS/gateway:$TRAVIS_TAG;
-
-        docker tag $DOCKER_NS/gateway:latest-dev quay.io/$DOCKER_NS/gateway:$TRAVIS_TAG;
         echo $QUAY_PASSWORD | docker login -u=$QUAY_USERNAME --password-stdin quay.io;
-        docker push quay.io/$DOCKER_NS/gateway:$TRAVIS_TAG;
-
-        docker tag $DOCKER_NS/classic-watchdog:latest-dev-armhf $DOCKER_NS/classic-watchdog:$TRAVIS_TAG-armhf;
-        docker tag $DOCKER_NS/classic-watchdog:latest-dev-arm64 $DOCKER_NS/classic-watchdog:$TRAVIS_TAG-arm64;
-        docker tag $DOCKER_NS/classic-watchdog:latest-dev-windows $DOCKER_NS/classic-watchdog:$TRAVIS_TAG-windows;
-        docker tag $DOCKER_NS/classic-watchdog:latest-dev-x86_64 $DOCKER_NS/classic-watchdog:$TRAVIS_TAG-x86_64;
-        echo $DOCKER_PASSWORD | docker login -u=$DOCKER_USERNAME --password-stdin;
-        docker push $DOCKER_NS/classic-watchdog:$TRAVIS_TAG-armhf;
-        docker push $DOCKER_NS/classic-watchdog:$TRAVIS_TAG-arm64;
-        docker push $DOCKER_NS/classic-watchdog:$TRAVIS_TAG-windows; 
-        docker push $DOCKER_NS/classic-watchdog:$TRAVIS_TAG-x86_64;
-
-        docker tag $DOCKER_NS/classic-watchdog:latest-dev-armhf quay.io/$DOCKER_NS/classic-watchdog:$TRAVIS_TAG-armhf;
-        docker tag $DOCKER_NS/classic-watchdog:latest-dev-arm64 quay.io/$DOCKER_NS/classic-watchdog:$TRAVIS_TAG-arm64;
-        docker tag $DOCKER_NS/classic-watchdog:latest-dev-windows quay.io/$DOCKER_NS/classic-watchdog:$TRAVIS_TAG-windows;
-        docker tag $DOCKER_NS/classic-watchdog:latest-dev-x86_64 quay.io/$DOCKER_NS/classic-watchdog:$TRAVIS_TAG-x86_64;
+        
+        ./ci/tagAndPush.sh "$DOCKER_NS/gateway";
```

Comment:
I like the new helper script. I would be keen to see the quay / hub pushing at a higher level and passed down.

I've merged the changes and ma doing a release now with an arm build.

Is there a way that it makes sense to keep the docker login one level higher than this?

```
         REGISTRY=docker.io
        ./ci/login.sh docker.io
        ./ci/tagAndPush.sh $REGISTRY "$DOCKER_NS/gateway"

        REGISTRY=quay.io
        ./ci/login.sh quay.io
        ./ci/tagAndPush.sh $REGISTRY "$DOCKER_NS/gateway"

```

Reply:
I'm easy.  The motivation was to reduce the verbosity of (and repetition in) `.travis.yml`.  This will reduce the gains in that regard. #1234 is available should you wish to take it.